frontend/public/components/factory/list.tsx

```diff
@@ -194,6 +195,32 @@ const listFilters = {
   },
 };
 
+/* Append filters from plugins
+
+TODO(mlibra):
+Some objects (like VirtualMachines) require to query additional objects
+to fully asses state (like pods and CRs in the case of a VirtualMachine).
+Recent filtering logic can handle just a single "listed" object.
+We will need to find a way how to pass additional resources there.
+*/
+(function () {
+  const baseListCount = Object.keys(listFilters).length;
+
+  plugins.registry.getResourceListPages()
+    .filter(rlp => !!rlp.properties.filter)
+    .forEach(rlp => {
+      if (listFilters[rlp.properties.model.kind]) {
+        // eslint-disable-next-line no-console
+        console.warn(`List filter for ${rlp.properties.model.kind} already defined, skipping.`);
+      } else {
+        listFilters[rlp.properties.model.kind] = rlp.properties.filter;
+      }
+    });
+
+  // eslint-disable-next-line no-console
+  console.info(`${Object.keys(listFilters).length - baseListCount} new list filters added by plugins.`);
```

Comment:
Is this really necessary?

Reply:
This is in-line with a former convention to inform users about extensions being applied & any potential conflicts.

With #1708 we no longer need this because
- conflicts are detected via extension checks (test phase)
- reporting on various extensions is meant to be done via `plugin-stats` (printed as part of the Console webpack build)